src/instantx.cpp

```diff
@@ -73,7 +73,10 @@ void CInstantSend::ProcessMessage(CNode* pfrom, const std::string& strCommand, C
 
         uint256 nVoteHash = vote.GetHash();
 
-        pfrom->setAskFor.erase(nVoteHash);
+        {
+            LOCK(cs_main);
+            connman.RemoveAskFor(CInv(MSG_TXLOCK_VOTE, nVoteHash));
```

Comment:
still uses `CInv`?

Reply:
Strange. Looks like I did the global search too early after branch switching (CLion indexes out of date maybe). Fixing in a squashed commit.